tests/integration/test_advanced_indexing.py

```diff
@@ -756,6 +756,15 @@ def test():
     res_num = x_num[[1, 1], :, [False, True, False, True]]
     assert np.array_equal(res, res_num)
 
+    # set item with mixed indices
+    x[1, :, [False, True, False, True]] = 129
+    x_num[1, :, [False, True, False, True]] = 129
+    assert np.array_equal(x, x_num)
+
```

Comment:
Can you also add a case where there's items after the boolean array? e.g. `x[:, [False, True, False, True], 1]`